modules/javafx.graphics/src/main/java/javafx/scene/Parent.java

```diff
@@ -1926,7 +1926,7 @@ void releaseAccessible() {
     /**
      * Note: The only user of this method is in unit test: Parent_structure_sync_Test.
      */
-    List<Node> test_getRemoved() {
+    public List<Node> test_getRemoved() {
```

Comment:
This is not an acceptable change. We do not add public API to support tests. Take a look at the Shim classes for an example of how to handle this.

Reply:
Done